src/gui/addnewtorrentdialog.cpp

```diff
@@ -175,6 +178,35 @@ void AddNewTorrentDialog::setTopLevel(bool value)
     SettingsStorage::instance()->storeValue(KEY_TOPLEVEL, value);
 }
 
+int AddNewTorrentDialog::savePathHistoryLength()
+{
+    return savePathHistoryLengthSetting();
+}
+
+namespace
+{
+    const int minPathHistoryLength = 0;
```

Comment:
I would prefer to keep these constants in usual place (near the others).

Reply:
Moved.